### ISSUE TITLE
Add CI for building on ARM(64).

### ISSUE THREAD

**Issue by christianparpart** [contour-terminal/contour#255]
~~I don't have one, but~~ it would be nice to ensure via CI that Contour's building on ARM64. Maybe we can even provide binary release builds then.

Ideally Linux/ARM, but I think maybe there's OSX/ARM or so available?

**Comment by fleverest** [comment]
Seems to build successfully on WSL2, but can't test without the GUI yet :)

**Comment by fleverest** [comment]
Yes, that’s one of my development environments - happy to test when I can!

**Comment by uspasojevic96** [comment]
we can re-enable archlinux arm builds on CI

**Comment by christianparpart** [comment]
> we can re-enable archlinux arm builds on CI

oh yes, right. I remember. but we disabled because it was way to slow. Maybe we find a more performant way *or* we don't make it as a requirement for merges. I don't think Github Actions does have native ARM64 support on any OS, do they (have to check...).

**Comment by uspasojevic96** [comment]
it was being done through qemu

**Comment by christianparpart** [comment]
I now naught the Raspberry Pi 400, which is an ARM64 with all you need tiny PC inside a keyboard. I can't await testing contour live on stream on that tiny computer (p.s. we need to make sure we only require OpenGL ES 3.1 but Contour currently requires ES 3.2 (not sure if I need to change actual code for that).

**Comment by WSLUser** [comment]
I'd suggest looking at what VTE (for Linux), Windows Terminal (for Windows) and Iterm2 do for their ARM64 builds.

**Comment by uspasojevic96** [comment]
We need to maintain Arch Linux and Ubuntu builds due to current the userbase, iTerm2 can easily build ARM64 because of Apple Silicon and because it's macOS only. Keep in mind we need to cover Arch, Ubuntu, Windows, macOS builds at least for ARM64 and we had only Arch to begin with. I don't see how feasible will it be to do single build for multiple platforms through macOS (if possible, can you share some links?)

**Comment by Yaraslaut** [comment]
Done
